src/main/resources/com/google/api/codegen/php/standalone_sample.snip

```diff
@@ -14,7 +14,7 @@
      */
 
     // [START {@sample.regionTag}]
-    namespace {@sampleFile.fileHeader.packageName};
+    require __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
We shouldn't hardcode the depth of the directories here. The 'vendor' directory will be alongside the generated composer.json file, so we should calculate this based on the output directory of composer.json vs the sample file.